src/content/basics/nodepools/index.md

```diff
@@ -139,6 +139,14 @@ Just as the Giant Swarm API schema for v4 (without node pools) and v5 (with node
 The new definition schema for v5 allows for defining cluster and node pool details in one file,
 to be submitted for creation via the [`gsctl create cluster`](/reference/gsctl/create-cluster/) command.
 
+## Node pools and autoscaling {#autoscaling}
+
+With node pools, you set the autoscaling range per node pool. The Kubernetes cluster autoscaler has to decide which node pool to scale under which circumstances.
+
+If you assign workloads to node pools as described [above](#assigning-workloads) and the autoscaler finds pods that are not assigned to any node, it will decide based on the node selectors which node pools to scale up.
```

Comment:
```suggestion
If you assign workloads to node pools as described [above](#assigning-workloads) and the autoscaler finds pods in `Pending` state, it will decide based on the node selectors which node pools to scale up.
```